adr/009-docker.md

```diff
@@ -0,0 +1,25 @@
+# 9. Docker Containerization
+
+Date: 2022-01-04
+
+## Decision
+
+We will use [Docker](https://docs.docker.com/get-started/overview/) for containerization of our application.
+
+## Status
+
+Accepted.
+
+## Context
+
+Docker is an open platform for developing, shipping, and running applications. Docker provides the ability to package and run an application in a loosely isolated environment. Some benefits of using Docker:
+- Develop your application and its supporting components using containers.
+- The container becomes the unit for distributing and testing your application.
+- When you’re ready, deploy your application into your production environment, as a container or an orchestrated service.
+- Allows for responsive deployment and scaling.
+
+[Reference](https://docs.docker.com/get-started/overview/)
+
+### Related Issues
+
+- [story/76/Containerize application with docker](https://app.zenhub.com/workspaces/cdc-ti-633de70d939f6c00155f3722/issues/gh/cdcgov/trusted-intermediary/76)
```

Comment:
```suggestion
- #76
```

I'd recommend not using a ZenHub link since that is internal to just us.  That link won't work for others.

Reply:
Good call.